src/slurm_plugin/cluster_event_publisher.py

```diff
@@ -576,12 +712,25 @@ def callable_event_publisher(
 
                         event_logger.log(event_level, "%s", json.dumps(dict(event)))
                     except Exception as e:
-                        logger.error("Failed to publish event: %s\n%s", e, traceback.format_exception(*sys.exc_info()))
+                        extraction = traceback.extract_stack()
+                        logger.error(
+                            "Failed to publish event `%s`: %s\n%s\n%s",
+                            event_type,
+                            e,
+                            "".join(traceback.format_exception(*sys.exc_info())),
+                            "".join(traceback.format_list(extraction)),
+                        )
 
         return callable_event_publisher
 
+    @staticmethod
+    def _is_idle_node(node: SlurmNode) -> bool:
```

Comment:
Looks like this is a methods of the SlurMNode class, what are moving the method into SlurmNode class?

Reply:
Moved.